pkgs/kde/third-party/wallpaper-engine-plugin/default.nix

```diff
@@ -0,0 +1,60 @@
+{
+  extra-cmake-modules,
+  fetchFromGitHub,
+  full,
+  kpackage,
+  libplasma,
+  lib,
+  lz4,
+  mkKdeDerivation,
+  mpv,
+  pkg-config,
+  python3,
+}:
+mkKdeDerivation {
+  pname = "wallpaper-engine-kde-plugin";
+  version = "qt6";
```

Comment:
Please see the contributing guide on what we should fill in here.

Reply:
I've updated the version number here